ui/package.json

```diff
@@ -135,7 +135,7 @@
   "dependencies": {
     "@influxdata/clockface": "2.3.4",
     "@influxdata/flux": "^0.5.1",
-    "@influxdata/flux-lsp-browser": "^0.5.31",
+    "@influxdata/flux-lsp-browser": "^0.5.41",
```

Comment:
Meta: I don't think these changes actually do anything now that we pull from the `ui` repo, we should tell the Flux team they don't need to bump this any more (unless it's needed for `e2e` to pass?)

@wbaker85 can you make sure our tag in the UI repo includes this newest version?

Reply:
UI repo release for 2.0.5 is updated with the newest flux version!

As far as if the updates are needed for our `e2e` tests to pass...I *doubt* it, but I don't know for sure. I'm going to really push for getting the `e2e` testing moved over to `monitor-ci` in the next few days so at that point we can definitely stop updating `flux-lsp-browser` here (and just delete all the UI source entirely).